src/scene_battle_rpg2k3.cpp

```diff
@@ -508,6 +508,7 @@ void Scene_Battle_Rpg2k3::RefreshTargetWindow() {
 	// FIXME: Handle live refresh in traditional when the window is always visible
 	auto commands = GetEnemyTargetNames();
 	target_window->ReplaceCommands(std::move(commands));
+	if (!target_window->GetActive()) target_window->SetIndex(-1);
```

Comment:
Missing {} and indent.

Can your change cause any race condition? E.g. the target window is open and while open an enemy dies (if this is even possible).
Does this fix any known bug?

Reply:
Fixed the missing {} and indent. This change fixes the cursor being shown even if the target window is not active. In RPG_RT the cursor is only shown in the target window if you are selecting a target. If the target window is open the active flag should always be set which means that the case you mentioned should never occur.